R/twas.R

```diff
@@ -502,7 +502,7 @@ twas_weights <- function(X, Y, weight_methods, num_threads = 1, seed = NULL) {
   if (num_cores >= 2) {
     # Set up parallel backend to use multiple cores
     plan(multicore, workers = num_cores)
-    weights_list <- names(weight_methods) %>% future_map(compute_method_weights, weight_methods, .options = furrr_options(seed = seed))
+    weights_list <- names(weight_methods) %>% future_map(compute_method_weights, weight_methods, .options = furrr_options(seed=TRUE))
```

Comment:
in the previous call of `furrr_options` you have `globals` but you don't have it here? Please double check to make sure this is intended.

Reply:
yes, this step is for twas weight computation without cv 
`twas_weights_res <- twas_weights(X = X, Y = Y, weight_methods = weight_methods)` 
future achieve it through the function 
`compute_method_weights <- function(method_name, weight_methods) `
so there are no extra parameters needed to be introduced other than weight_methods, 

The one above which has globals parameters was for twas_cv, that some additional argument will need to be included (such as sample_partition,  args, cv_args) as the facilitating function is only asking for fold number but it would need additional information from the environment. 
`compute_method_predictions <- function(j){...}`